llm/server.go

```diff
@@ -671,71 +682,34 @@ type ImageData struct {
 	AspectRatioID int    `json:"aspect_ratio_id"`
 }
 
-type completion struct {
-	Content      string `json:"content"`
-	Model        string `json:"model"`
-	Prompt       string `json:"prompt"`
-	Stop         bool   `json:"stop"`
-	StoppedLimit bool   `json:"stopped_limit"`
-
-	Timings struct {
-		PredictedN  int     `json:"predicted_n"`
-		PredictedMS float64 `json:"predicted_ms"`
-		PromptN     int     `json:"prompt_n"`
-		PromptMS    float64 `json:"prompt_ms"`
-	}
```

Comment:
does this map to `time.Duration`?

Reply:
and does it match back to the same format when parsed?

Reply:
Yup! We had a bunch of roundabout logic that eventually arrived at a time.Duration:
https://github.com/ollama/ollama/pull/9324#discussion_r1968596091